modules/oc-mirror-creating-image-set-config.adoc

```diff
@@ -7,19 +7,20 @@
 [id="oc-mirror-creating-image-set-config_{context}"]
 = Creating the image set configuration
 
-Before you can use the oc-mirror plugin to mirror image sets, you must create an image set configuration file. This image set configuration file defines which {product-title} releases, Operators, and other images to mirror, along with other configuration settings for the oc-mirror plugin.
-
-You must specify a storage backend in the image set configuration file. This storage backend can be a local directory or a registry that supports link:https://docs.docker.com/registry/spec/manifest-v2-2[Docker v2-2]. The oc-mirror plugin stores metadata in this storage backend during image set creation.
-
-[IMPORTANT]
-====
-Do not delete or modify the metadata that is generated by the oc-mirror plugin. You must use the same storage backend every time you run the oc-mirror plugin for the same mirror registry.
-====
+In order to use the oc-mirror plugin to mirror image sets, you must create an image set configuration file. This image set configuration file defines which {product-title} releases, Operators, and other images to mirror, along with other configuration settings for the oc-mirror plugin.
 
 .Prerequisites
 
 * You have created a container image registry credentials file. For instructions, see _Configuring credentials that allow images to be mirrored_.
 
+* oc-mirror has the following permissions: 
+** To push images to the mirror registry
+*** Images of the imageset corresponding to the declared image set configuration
```

Comment:
This line is confusing actually also did not really understand what you mean here.